src/data-services/ce-backend.js

```diff
@@ -59,6 +59,42 @@ function getData(
   });
 }
 
+// TODO: https://github.com/pacificclimate/climate-explorer-frontend/issues/124
```

Comment:
Isn't this TODO what you just did here? Or are there more waiting to be extracted?